distribution/src/config/log4j2.properties

```diff
@@ -29,58 +31,144 @@ appender.rolling.strategy.action.condition.type = IfFileName
 appender.rolling.strategy.action.condition.glob = ${sys:es.logs.cluster_name}-*
 appender.rolling.strategy.action.condition.nested_condition.type = IfAccumulatedFileSize
 appender.rolling.strategy.action.condition.nested_condition.exceeds = 2GB
+##############################################
+######## Server -  old style pattern ########
```

Comment:
Very minor nit: Is it intentional that the comments for the old style are different than for JSON? Also, is it intentional that the first commen line has a different length than the second one?

Reply:
I will realign them, these should be same length 50.